Engine/src/Engine/ResourceManagement/Audio.cpp

```diff
@@ -15,8 +15,8 @@ AudioPlayerSingleton* AudioPlayerSingleton::getInstance()
 	if (m_instance == nullptr)
 	{
 		m_instance = new AudioPlayerSingleton();
-		m_engineMediaPath = std::filesystem::current_path().parent_path().string() + "\\Engine\\src\\Media\\";
-		m_gameMediaPath = std::filesystem::current_path().parent_path().string() + "\\Game\\src\\Media\\";
+		m_engineMediaPath = std::filesystem::current_path().parent_path().string() + "\\Engine\\src\\Engine\\Media\\";
```

Comment:
there is a cmake command that involve some of these path; just double check the debug.exe and release.exe are still working properly

Reply:
Just checked, they do.